apps/meteor/tests/e2e/page-objects/omnichannel-info.ts

```diff
@@ -26,4 +26,8 @@ export class OmnichannelContactInfo extends OmnichannelManageContact {
 	get historyMessage(): Locator {
 		return this.dialogContactInfo.getByRole('listitem').first();
 	}
+
+	get openChatBtn(): Locator {
+		return this.dialogContactInfo.getByRole('button', { name: 'Open chat' });
+	}
```

Comment:
Is this new locator being used?
```suggestion
```

Reply:
Nope, it was leftovers from a test that I've refactored, removing it now